tests/end2end/features/tabs.feature

```diff
@@ -1026,3 +1026,100 @@ Feature: Tab management
             - tabs:
               - history:
                 - url: http://localhost:*/data/hello.txt
+
+    # :tab-pin
+
+    Scenario: :tab-pin command
+        When I open data/numbers/1.txt
+        And I open data/numbers/2.txt in a new tab
+        And I open data/numbers/3.txt in a new tab
+        And I run :tab-pin
```

Comment:
I would like to have a 'Then tab 2 is pinned' statement here, or something along the lines of that. Where would that be implemented? (Haven't worked too much with this style of testing before, sorry :P)